src/tests/useQuery.test.js

```diff
@@ -998,4 +998,71 @@ describe('useQuery', () => {
 
     await waitForElement(() => rendered.getByText('rendered'))
   })
+
+  it('should update data upon interval changes', async () => {
+    let count = 0
+    function Page() {
+      const [int, setInt] = React.useState(200)
+      const { data } = useQuery('/api', () => count++, {
+        refetchInterval: int,
+      })
+      return (
+        <div onClick={() => setInt(num => (num < 400 ? num + 100 : 0))}>
+          count: {data}
+        </div>
+      )
+    }
+    const { container } = render(<Page />)
+    expect(container.firstChild.textContent).toMatchInlineSnapshot(`"count: "`)
```

Comment:
Is there any way to do this without snapshots? I'm not opposed to them if they are indeed the best choice, but when possible, I try to avoid them.